tests/jballerina-unit-test/src/test/resources/test-src/types/constant/list_constructor_expr_as_constant_expr.bal

```diff
@@ -0,0 +1,330 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+const int[2] ARR_11 = [3, 1];
```

Comment:
Do we have test cases covering these?

```
const arr = [1, 2, 3];
```

Reply:
Yes. We have test cases without `type node` for list constant
https://github.com/ballerina-platform/ballerina-lang/pull/40621/files#diff-d0511fbe8989ff73a4f920558e0e0e01c4905d164e66532fd35cfa1da119c888R268